db.json

```diff
@@ -441,6 +441,27 @@
         }
       ],
       "standards": [
+        {
+          "name": "Compression",
+          "href": "https://compression.spec.whatwg.org/",
+          "description": "The Compression Standard provides APIs for compressing and decompressing bytes using a set of the most common compression algorithms.",
+          "authors": [
+            {
+              "name": "Adam Rice",
+              "email": "ricea@chromium.org"
+            },
+            {
+              "name": "Canon Mukai",
+              "email": "canonmukai@google.com"
+            }
+          ],
+          "reference": "COMPRESSION",
+          "review_draft_schedule": [
+            2,
+            8
```

Comment:
```suggestion
            4,
            10
```

Reply:
I'm merging this, but I have no idea how is this decided.